app/services/mailer/transactional_email.rb

```diff
@@ -0,0 +1,76 @@
+require "mandrill"
+
+class TransactionalEmail
+  attr_reader :mandrill, :template_name, :brand, :merge_params, :message
+
+  module Types
+    WELCOME     = 'welcome'
+    PLAN_CHANGE = 'plan-change'
+    CANCEL_PLAN = 'cancel-plan'
+  end
+
+  def initialize(template_name, brand)
+    @mandrill = Mandrill::API.new(ENV["MANDRILL_API_KEY"])
```

Comment:
Can move this into an initializer and have a constant for this? Should only need to instantiated once.